unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatWindowComponentView.cs

```diff
@@ -38,27 +56,72 @@ public static PublicChatWindowComponentView Create()
     public override void Awake()
     {
         base.Awake();
+
         backButton.onClick.AddListener(() => OnBack?.Invoke());
         closeButton.onClick.AddListener(() => OnClose?.Invoke());
         muteToggle.OnSelectedChanged += (b, s, arg3) => OnMuteChanged?.Invoke(b);
+        expandMembersListButton.onClick.AddListener(ToggleMembersSection);
+        collapseMembersListButton.onClick.AddListener(ToggleMembersSection);
+        collapsableAreaOriginalWidth = collapsableArea.sizeDelta.x;
+        membersIconButton.onClick.AddListener(ToggleMembersSection);
+        targetGraphicColor = membersIconButton.targetGraphic.color;
+
+        if (goToCrowdButton != null)
+            goToCrowdButton.onClick.AddListener(() => OnGoToCrowd?.Invoke());
     }
-    
+
     public override void RefreshControl()
     {
         nameLabel.text = $"~{model.name}";
+        memberCountLabel.text = model.memberCount.ToString();
         muteToggle.SetIsOnWithoutNotify(model.muted);
     }
 
     public void Hide() => gameObject.SetActive(false);
 
-    public void Show() => gameObject.SetActive(true);
-    
+    public void Show()
+    {
+        gameObject.SetActive(true);
+
+        if (!isMembersSectionOpen)
+            ToggleMembersSection();
+    }
+
     public void Configure(PublicChatModel model)
     {
         this.model = model;
         RefreshControl();
     }
-    
+
     public void OnPointerDown(PointerEventData eventData) => OnClickOverWindow?.Invoke();
-  
-}
\ No newline at end of file
+
+    public void UpdateMembersCount(int membersAmount) =>
+        memberCountLabel.text = membersAmount.ToString();
+
+    private void ToggleMembersSection()
+    {
+        isMembersSectionOpen = !isMembersSectionOpen;
+
+        expandMembersListButton.gameObject.SetActive(!isMembersSectionOpen);
+        collapseMembersListButton.gameObject.SetActive(isMembersSectionOpen);
+
+        collapsableArea.sizeDelta = new Vector2(
```

Comment:
It's not necessary to change but: 
`sizeDelta`'s meaning is dependent on anchors : e.g. if stretched they act as padding. It is safer to use `SetSizeWithCurrentAnchors` to ensure it will work for every anchors set.